lib/system.nim

```diff
@@ -2648,19 +2648,40 @@ proc compiles*(x: untyped): bool {.magic: "Compiles", noSideEffect, compileTime.
   ##     echo "'+' for integers is available"
   discard
 
+proc isNamedTuple*(T: type): bool =
```

Comment:
Can't we avoid adding this to system.nim? It could be moved to `helpers.nim` and exported in `typetraits`, `sugar` or some other fitting module.

Reply:
fixed